roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: install dependencies
+  apt:
+    pkg: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - ca-certificates
+    - curl
+    - python-pip
+    - unattended-upgrades
+- name: add apt key for docker
+  apt_key:
+    data: "{{ lookup('file', 'docker.asc') }}"
+    state: present
+    keyring: /etc/apt/trusted.gpg.d/docker.gpg
+- name: add apt repository for docker
+  apt_repository:
+    filename: docker
+    state: present
+    update_cache: yes
+    repo: "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+- name: install docker-ce
+  apt:
+    pkg: docker-ce
+    state: present
+- name: install docker python bindings
+  pip:
+    name: docker-py
+    state: present
+- name: add user vagrant to docker group
+  user:
+    name: vagrant
+    groups: docker
+    append: yes
+- name: create custom shell for jumping to docker
+  copy:
+    src: dockershell
+    dest: /bin/dockershell
+    mode: 0755
+    owner: root
+    group: root
+#- name: add dockershell as valid shell
```

Comment:
Do you really mean to keep this commented?

Reply:
Removed.